AK/IPv6Address.h

```diff
@@ -51,25 +51,24 @@ class [[gnu::packed]] IPv6Address {
 #ifdef KERNEL
     ErrorOr<NonnullOwnPtr<Kernel::KString>> to_string() const
 #else
-    DeprecatedString to_deprecated_string() const
+    ErrorOr<String> to_string() const
 #endif
     {
         if (is_zero()) {
 #ifdef KERNEL
             return Kernel::KString::try_create("::"sv);
 #else
-            return "::"sv;
+            return String::from_utf8_short_string("::"sv);
```

Comment:
I think this would be more appropriate? Haven't tested it, but I think that's how it's written.
```suggestion
            return "::"_short_string;
```

Reply:
Ah, nice. Seems to work great :)